habit/src/store/configure-store.js

```diff
@@ -0,0 +1,19 @@
+import { createStore, applyMiddleware, compose } from 'redux'
+import thunk from 'redux-thunk'
+import { logger } from 'redux-logger'
+import rootReducer from '../reducers/root-reducer'
+
+const middleware = [thunk];
```

Comment:
We should re-think about logger.
If not using it, do something like
```
const store = createStore(rootreducer, applyMiddleware(thunk));

export default store;
```

Reply:
yeah, removing logger